2d/benchmarks/dambreak_Colagrossi/helpers.py

```diff
@@ -1,12 +1,14 @@
 def CreateFig():
     from tables import  openFile
-    archive = openFile('dambreak_Colagrossi0.h5','r')
+    archive = openFile('dambreak_Colagrossi_p.h5','r')
     import dambreak_Colagrossi
     import dambreak_Colagrossi_so
     import matplotlib.tri as mtri
     from matplotlib import pyplot as  plt
     import numpy as np
     domain = dambreak_Colagrossi.domain
+    domain.L = dambreak_Colagrossi.tank_dim
```

Comment:
I see. Is assembleDomain not called in the problem module? dambreak_Colagrossi.py in this case? The bounding box in the form of x and L should be available once the domain has been contructed, but it's not in this dambreak cases so we're having to use the above work around. That's what I'm tying to get rid of.